tests/e2e/tidbcluster/serial.go

```diff
@@ -912,3 +952,15 @@ var _ = ginkgo.Describe("[tidb-operator][Serial]", func() {
 		})
 	})
 })
+
+func setPartitionAnnotation(namespace, tcName, component string, ordinal int) error {
+	// add annotation to pause statefulset upgrade process
+	cmd := fmt.Sprintf("kubectl annotate tc %s -n %s tidb.pingcap.com/%s-partition=%d --overwrite",
+		tcName, namespace, component, ordinal)
+	klog.Infof("%s", cmd)
+	output, err := exec.Command("/bin/sh", "-c", cmd).CombinedOutput()
+	if err != nil {
+		return fmt.Errorf("fail to set annotation for [%s/%s], component: %s, partition: %d, err: %v, output: %s", namespace, tcName, component, ordinal, err, string(output))
+	}
+	return nil
+}
```

Comment:
use `framework.RunKubectl` or [related functions](https://github.com/kubernetes/kubernetes/blob/68ccd2e7129b9e7064e353d3dd05fcff83d5afc3/test/e2e/framework/util.go#L1557-L1591) to run `kubectl` command, e.g. 

https://github.com/pingcap/tidb-operator/blob/46ad1990137adbd7ea679aeca37de93879a55afd/tests/e2e/tidbcluster/tidbcluster.go#L1223-L1232

Reply:
updated.